blazor/datagrid/pdf-export.md

```diff
@@ -1183,6 +1183,76 @@ In the following code sample, we have customized the column width for the PDF ex
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-customize-column-in-pdf-exported-document).
 
+### Export Date format in Grid header
```

Comment:
already we have documentation for pdf. need to draft the same for excel export only. https://blazor.syncfusion.com/documentation/datagrid/excel-exporting

Reply:
moved this topic to excel export.